google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1166,6 +1172,10 @@ public SpannerOptions build() {
         // As we are using plain text, we should never send any credentials.
         this.setCredentials(NoCredentials.getInstance());
       }
+      if (this.numChannels == null) {
+        this.numChannels = DEFAULT_CHANNELS;
```

Comment:
this.num
```suggestion
        this.numChannels = this.grpcGcpExtensionEnabled ? GRPC_GCP_ENABLED_DEFAULT_CHANNELS : DEFAULT_CHANNELS;
```

Reply:
Thanks for the suggestion @olavloite. This makes it more readable than the previous one.
I have made the changes in the recent commit.